src/playbook.yaml

```diff
@@ -95,3 +95,11 @@
 
     - name: Install BiocManager
       shell: "R -e \"install.packages('BiocManager')\""
+
+    - name: update limits.conf
+      shell:
+        cmd:
+          # The ">>" commands won't work unless we make this permission change
+          sudo chmod 666 /etc/security/limits.conf
```

Comment:
maybe we should restore the file permission after the operation?

Reply:
Its better to not change the permissions on this file at all, but instead use sudo to run the next two commands. I think this is because packer is building the AMI as the ec2-user, so to do actions as root, just sudo.

Reply:
@zaro0508 Yes, I can change the permissions back.  I will update the PR.  @ahayden That's a good idea but it didn't work when I tried it on an RStudio instance.  I.e., I ran "sudo echo ... >> /etc/security/limits.conf" and it didn't work.  But once I changed the permissions the command *did* work (without 'sudo', i.e. as ssmuser).